x-pack/plugins/siem/public/components/timeline/properties/helpers.tsx

```diff
@@ -156,10 +158,21 @@ const LargeNotesButton = pure<{ noteIds: string[]; text?: string; toggleShowNote
       onClick={() => toggleShowNotes()}
       size="l"
     >
-      <NotesButtonLabel>
-        <NotesIcon noteIds={noteIds} size="l" />
-        {text && text.length ? <LabelText>{text}</LabelText> : null}
-      </NotesButtonLabel>
+      <EuiFlexGroup alignItems="center" gutterSize="none" justifyContent="center">
+        <EuiFlexItem grow={false}>
+          <EuiIcon color="subdued" size="m" type="editorComment" />
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <NotesButtonLabel>
```

Comment:
Good use of `EuiFlexGroup` and `EuiFlexItem.`

Under the covers `NotesButtonLabel` is:

```
export const NotesButtonLabel = styled.div`
  align-items: center;
  justify-content: space-between;
  display: flex;
`;

I think at this point we should be able to remove these flex elements since are now moving to `EuiFlexItem`.